### PR TITLE
Discussion core addon

### PR DESCRIPTION
**Pull request by jensens**
see https://github.com/plone/plone.app.discussion/pull/211 and do not merge alone

### CODE REVIEW

**Comment by mauritsvanrees** [comment]
This could be continued on for Plone 6.1.
The `Plone` master branch already depends on `plone.app.discussion`, so that part of turning it into a core add-on is done already.

**Comment by mauritsvanrees** [comment]
I have just made a release 6.1.0a1 of the `Plone` package, where `plone.app.discussion` was added in the requirements, making it a core add-on.
But it is still a requirement of `Products.CMFPlone`, so not actually a core add-on. :-/. Oh well.
Would be good to continue this draft PR.

**Comment by jensens** [comment]
Indeed, I just had no time to follow up here. Let's see I I can make it happen next 1-2 weeks.

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Review by stevepiercy — APPROVED** [review]
Thanks for replacing the offensive terms.

File: news/3782.bugfix [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Turn plone.app.discussion in a core-addon.
```
Comment (davisagli) [inline news/3782.bugfix:1]:
```suggestion
plone.app.discussion is now a core add-on. It can be installed separately from Products.CMFPlone.
```

**Review by davisagli — APPROVED** [review]

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Review by mauritsvanrees — APPROVED** [review]
LGTM.

**Review by gforcada — APPROVED** [review]

**Comment by jensens** [comment]
I am not able to work on this here before Sept. 10th.
I tested everything but the upgrade, so if anyone have an spare hour to test it (and another to fix it if needed), this would be awesome.

**Pull request merged**